html-forms/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+    <p><label>Email:</label></p>
+    <input type="text" name="email">
+    <p><label>Why is coding cool?</label></p>
+    <select name="Why is coding cool">
+    <option value="Because it's fun"><label for="Because it's fun">Because it's fun</label></option>
+    <option value="Because it's interesting"><label for= "Beacause it's interesting">Because it's interesting</label></option>
+    <option value="Because it's powerful"><label for="Because it's powerful">Because it's powerful</label></option>
+    </select>
+    <p><label>Favorite Language (choose one)</label></p>
+    <input type="radio" name="language" value="HTML"/><label for="html">HTML</label>
+    <input type="radio" name="language" value="CSS"/><label for="css">CSS</label>
+    <input type="radio" name="language" value="Javascript"/><label for="javascript">Javascript</label>
```

Comment:
These `label` elements are not configured correctly to match with their `input`s. Take a look at the documentation again, and pay close attention to mentions of the `for` and `id` attributes as related to label elements.